content/md/assets/about-product-link-rules.md

```diff
@@ -0,0 +1,92 @@
+
+## Product value assignment
+
+Once you have chosen and selected the products you want to apply the rule on, it is time to think about where, in the products, you want to assign those assets.
+
+This is done in the second part of the product link rule, in the `assign_assets_to` property. Thanks to this property, you will define to which product value you want to assign your assets. In other words, which attribute, locale and scope of the products you want to link your assets to. You can also decide whether you want to **add** new assets or **replace** the existing ones inside this product attribute.
+
+# Asset values extrapolation
+"Asset values extrapolation"? What the hell is that? It's the clever naming I just came up with to explain one last thing about the product link rule.
+
+You now know how to [select your products](#product_selection), and then [assign your assets](#product_value_assignment) in the right product attribute. That's a good start.
+
+But in some cases, you might need the rule to be a bit more powerful so it answers your expectations.
+As an example is better than 10,000 words, let's imagine this situation.
+
+## How does it work?
+You put all your user guides in the same asset family, called `user_instructions`.
+On the one hand, you have an asset of this family, let's say the user guide for a particular TV, the `XMLD500 TV`. This user guide asset has the following code: `XMLD500_fr_FR_user_guide`. It is the French version of the user guide for this TV.  
+On the other hand, you have your TV which SKU is `XMLD500`. The product sheet of this TV has an asset collection attribute called `user_guides`. It is localizable.  
+Now, what you want is to automatically link the `XMLD500_fr_FR_user_guide` to the `XMLD500` product, in the right attribute on the right locale, ie the French locale.
+
+How do you do that? I'll tell you. You're gonna need "**asset value extrapolation**".
+
+"Asset value extrapolation" is a mechanism by which you can target specific asset values in various fields of the product link rule so that they can be interpreted depending on the asset it is currently linking.
+
+As a prerequisite for this rule to work, we would need **two new attributes** in the structure of our `user_instructions` asset family. One named `product_ref`, and the other `locale`.
+In our example, for our `XMLD500_fr_FR_user_guide` asset, we would store the values `XMLD500` into the `product_ref` attribute and `fr_FR` in the `locale` attribute.
+
+::: info
+Those two new attributes can be easily filled by using the API and a simple regular expression on the code of the asset, as both information, the product reference and the locale are already in the code.
+:::
+
+In fine, the PIM is going to select the product which SKU is `XMLD500` and assign the asset to the `user_instructions` product attribute on the `fr_FR` locale. Exactly what we wanted. 😉
```

Comment:
not sure what you are trying to say with "In fine"? Do you mean "As a result"?

Reply:
Yes

Reply:
I changed by "as a result". thanks :)